components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ApplicationBasicInfo.java

```diff
@@ -35,6 +35,7 @@ public class ApplicationBasicInfo implements Serializable {
     private String accessUrl;
 
     private User appOwner;
+    private String tenantDomain;
```

Comment:
Should we name this tenant domain or organization name?

Reply:
This attribute relates to the tenant domain corresponding to the tenant ID of the service provider in `SP_APPS` table.